src/logger/loggerTaskEx.c

```diff
@@ -222,7 +222,7 @@ void loggerTaskEx(void *params)
                 if (!g_loggingShouldRun && is_logging) {
                         logging_stopped();
                         const LoggerMessage logStopMsg = getLogStopMessage();
-#if defined(SDCARD_SUPPORT)
+#if SDCARD_SUPPORT
                         queue_logfile_record(&logStopMsg);
 #endif
                         queueTelemetryRecord(&logStopMsg);
```

Comment:
We're starting to get a sea of ifdefs... Should discuss the long term implications and ways to mitigate.